db/tables/100_image_tags.sql

```diff
@@ -1,6 +1,6 @@
 -- Set up table
 CREATE TABLE Image_Tags (
-    ImageTagId integer NOT NULL,
+    ImageTagId integer NOT NULL UNIQUE,
```

Comment:
What the process for inserting an image tag? I don't think the ImageTagId we get from VoTT will be unique. Should we just generate an Id at db insertion time?